downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -22,7 +22,7 @@ package com.here.ort.downloader.vcs
 import com.here.ort.downloader.WorkingTree
 import com.here.ort.model.VcsInfo
 import com.here.ort.model.VcsType
-import com.here.ort.spdx.LicenseFileMatcher
+import com.here.ort.utils.LicenseFileMatcher
```

Comment:
Would you mind adding a rationale to the commit message body?

Reply:
I'm squashing as you suggested.